lib/inspec/resources/sybase_session.rb

```diff
@@ -44,10 +44,18 @@ def query(sql)
       # try to get a temp path
       sql_file_path = upload_sql_file(sql)
 
+      current_shell = inspec.command("echo $SHELL")
```

Comment:
Just for posterity, I'll lament that we don't have a better way, or a better place, to detect this. But I think we can let this go for now.

Reply:
Yes. I will add TODO so that we can track this and improve.